packages/ava/src/utils/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './isType';
```

Comment:
Is it too early to put some utils in the root directory? @neoddish @BBSQQ

Reply:
I think it's fine.
other modules could find possible utils ASAP

Reply:
+1